ShopifySharp/Filters/ShopifyOrderFilter.cs

```diff
@@ -10,21 +10,21 @@ namespace ShopifySharp.Filters
     public class ShopifyOrderFilter : ShopifyListFilter
     {
         /// <summary>
-        /// The status of orders to retrieve. Default is <see cref="ShopifyOrderStatus.Any"/>.
+        /// The status of orders to retrieve. Known values are "open", "closed", "cancelled" and "any" (default).
         /// </summary>
         [JsonProperty("status")]
-        public ShopifyOrderStatus? Status { get; set; } = ShopifyOrderStatus.Any;
+        public string Status { get; set; } = "any";
```

Comment:
Maybe not default this to "any", as if the developer forgets to populate it, pulling any can be dangerous as it would pull all orders. with some of the shops I've been testing with I've found not all of them Archive their orders because they like visibility of them - defaulting this could cause hundreds of thousands of orders to potentially be pulled